src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,137 @@
+- Before cutover to the standby, the standby cluster does not support running [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) or [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}).
+- After a cutover, there is no mechanism to stop applications from connecting to the original primary. It is necessary to move over application traffic manually.
+
+{% comment %}this title needs updating{% endcomment %}
+## Get started
+
+This section is a quick overview of the initial requirements to start a replication stream.
+
+For more comprehensive guides, refer to:
+
+- [Technical Overview]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}): to understand physical cluster replication in more depth before setup.
+- [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}): for a tutorial on how to start a replication stream.
+- [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}): for detail on metrics and observability into a replication stream.
+- [Cut Over from a Primary Cluster to a Standby Cluster]({% link {{ page.version.version }}/cutover-replication.md %}): for a guide on how to complete a replication stream and cut over to the standby cluster.
+
+### Starting clusters
+
+To initiate physical cluster replication on clusters, you must [start]({% link {{ page.version.version }}/cockroach-start.md %}) the primary and standby CockroachDB clusters with the `--config-profile` flag. This enables cluster virtualization {% comment %}link to CV docs{% endcomment %} and sets up each cluster ready for replication.
+
+The active primary cluster that serves application traffic:
+
+~~~shell
+cockroach start ... --config-profile replication-source
+~~~
+
+The passive standby cluster that will ingest the replicated data:
+
+~~~shell
+cockroach start ... --config-profile replication-target
+~~~
+
+The node topology of the two clusters does not need to be the same. For example, you can provision the standby cluster with fewer nodes. However, keep the following in mind:
+
+- The standby cluster requires enough storage to contain the primary cluster's data.
+- During a failover scenario, the standby will need to handle the full production load. {% comment %}this should be in an include file and also added to the main tutorial{% endcomment %} However, the clusters cannot have different region topologies (refer to  [Limitations](#limitations)).
+
+{{site.data.alerts.callout_info}}
+Every node in the standby cluster must be able to make a network connection to every node in the primary cluster to start a replication stream successfully.
```

Comment:
Is it worth linking to the section from the task PR about copying the CA certificates?

Reply:
Done, good idea.